crates/sparrow-main/tests/e2e/cast_tests.rs

```diff
@@ -37,7 +37,7 @@ async fn cast_data_fixture() -> DataFixture {
 
 #[tokio::test]
 async fn test_implicit_cast_i64_to_f64_add() {
-    insta::assert_snapshot!(QueryFixture::new("{ i64_field: Input.i64, f64_field: Input.f64, add: Input.i64 + Input.f64 }").run_to_csv(&cast_data_fixture().await).await.unwrap(), @r###"
+    insta::assert_snapshot!(QueryFixture::new("{ add: Input.i64 + Input.f64 }").run_to_csv(&cast_data_fixture().await).await.unwrap(), @r###"
```

Comment:
how was this test passing? We seem to have removed fields from the output and yet we did not alter the expected result. Am I reading this wrong?

Reply:
This may behave been a "accidentally changed the wrong thing and saved". I don't believe this test is passing. Fixing.